lib/jxl/modular_test.cc

```diff
@@ -281,6 +282,75 @@ TEST(ModularTest, RoundtripLosslessCustomSqueeze) {
                                      /*distmap=*/nullptr, pool));
 }
 
+TEST(ModularTest, RoundtripLosslessSqueezeHighBitdepth) {
```

Comment:
I understand that we already do this in other tests, but I'd prefer if we made the test parametric instead of having an internal loop that makes debugging harder if a test does fail.

Reply:
Good point, I'll just have to check how to do that again.